ui/apos/components/AposDuplicateImportModal.vue

```diff
@@ -0,0 +1,331 @@
+          }
+        });
+      } catch (error) {
+        apos.notify(this.$t('aposImportExport:importCleanFailed'), {
+          type: 'warning',
+          dismiss: true,
+          interpolate: {
+            exportPath: this.exportPath
+          }
+        });
+      }
+    },
+    ready() {
+      this.$refs.runOverride.$el.querySelector('button').focus();
+    },
+    runOverride() {
+      try {
+        apos.http.post('/api/v1/@apostrophecms/import-export/override-duplicates', {
+          body: {
+            docIds: this.checked,
+            importedAttachments: this.importedAttachments,
+            exportPath: this.exportPath,
+            jobId: this.jobId,
+            notificationId: this.notificationId
+          }
+        });
+        this.exportFileCleaned = true;
+      } catch (error) {
+        apos.notify(this.$t('aposImportExport:exportFailed'), {
+          type: 'danger',
+          dismiss: true
+        });
+      }
+
+      this.modal.showModal = false;
+    },
+    async cancel() {
+      this.modal.showModal = false;
+    },
+    deselect() {
+      this.checked = this.checked.length
+        ? []
+        : this.duplicatedDocs.map(({ aposDocId }) => aposDocId);
+    }
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+.apos-import-duplicate {
+  z-index: $z-index-modal;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+::v-deep .apos-modal__inner {
+  top: auto;
+  right: auto;
+  bottom: auto;
+  left: auto;
+  height: auto;
+  text-align: left;
+}
+
+::v-deep .apos-modal__overlay {
+  .apos-modal+.apos-export & {
+    display: block;
+  }
+}
+
+::v-deep .apos-modal__body {
+  padding: 30px 20px;
+  width: 375px;
+}
+
+::v-deep .apos-modal__body-main {
+  display: flex;
+  flex-direction: column;
+  align-items: baseline;
+}
+
+::v-deep .apos-toggle__slider {
+  display: flex;
+}
+
+::v-deep .apos-input--select {
+  text-transform: capitalize;
+}
+
+.apos-import-duplicate__heading {
+  @include type-title;
+  line-height: var(--a-line-tall);
+  margin: 0;
+  text-transform: capitalize;
+}
+
+.apos-import-duplicate__description {
+  @include type-base;
+  font-size: var(--a-type-large);
+  text-align: left;
+  line-height: var(--a-line-tallest);
+  margin-top: 15px;
+  margin-bottom: 20px;
+  padding: 10px;
+  background-color: var(--a-warning-fade);
+  color: var(--a-warning-dark);
+}
+
+.apos-import-duplicate__section {
+  @include type-base;
+  display: flex;
+  flex-direction: column;
+  align-items: baseline;
+  min-width: 100%;
+  width: 100%;
+}
+
+.apos-import-duplicate__settings {
+  @include type-base;
+  font-weight: 600;
+  color: var(--a-base-3);
+  margin-top: 20px;
+}
+
+.apos-import-duplicate__docs-list {
+  width: 100%;
+  max-height: calc(60vh - 220px);
+  overflow-y: auto;
+  padding-bottom: 15px;
+}
+
+.apos-import-duplicate__separator {
+  background-color: var(--a-base-9);
```

Comment:
maybe configure our stylelint config

Reply:
Created a ticket for that since the existing apos config doesn't manage vue files